auth-sample/src/main/AndroidManifest.xml

```diff
@@ -30,7 +30,8 @@
         android:theme="@style/AppTheme">
         <activity
             android:name="com.spotify.sdk.android.auth.sample.MainActivity"
-            android:configChanges="keyboardHidden|keyboard|orientation|screenSize">
+            android:configChanges="keyboardHidden|keyboard|orientation|screenSize"
+            android:exported="true">
```

Comment:
Why do we need to export the MainActivity?

Reply:
Good question. I increased the `targetSdkVersion` to 31 also for sample app. Since Android 12 (SDK 31) it is required for activities to specify `exported` attribute. Because MainActivity is a launcher activity we need to specify `exported="true"`.